manifests/module_plugin.pp

```diff
@@ -71,24 +71,34 @@
     }
   }
 
+  $_owner = $owner ? {
```

Comment:
There may be a better way to do this pattern. I haven't done a ton of this style in Puppet.